docs/user/dql/functions.rst

```diff
@@ -2346,7 +2346,7 @@ Return type integer:
 
 Example::
 
-    os> SELECT POSITION('world' IN 'helloworld')
+    opensearchsql> SELECT POSITION('world' IN 'helloworld'), POSITION('invalid' IN 'helloworld');
```

Comment:
This should be `os>`. Tests with other prefixes are just ignored.

Reply:
Addressed in 5c78ab670f2fdb708b43eea0ede481e283f549de